tiledb/sm/c_api/tiledb.cc

```diff
@@ -822,12 +867,23 @@ int32_t tiledb_array_schema_load(
       return TILEDB_ERR;
     }
 
+    // With enumerations not supported for remote arrays currently.
```

Comment:
The two places where remote operations don't yet work are because I need to collaborate with the cloud REST team to add a query string parameter to one endpoint and then add an entirely new endpoint for the get_enumeration (after the schema was loaded) behavior. There are stories filed for that work but we decided they aren't enough to hold up merging this with the experimental tagging in the mean time since queries will work.